prism_benchmarks/run_prism_benchmarks.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# This script runs the benchmarks for the Prism parser in the Sorbet type checker.
+# It assumes the presence of a few directories and tools:
+# - The yjit-bench directory, which contains the benchmarks for YJIT (https://github.com/Shopify/yjit-bench)
+# - Hyperfine, a command-line benchmarking tool (https://github.com/sharkdp/hyperfine)
+
+# Check if the yjit-bench directory exists
+if [ ! -d "../yjit-bench" ]; then
+  echo "Please clone the yjit-bench directory before running this script: https://github.com/Shopify/yjit-bench"
+  exit 1
+fi
+
+if [! command -v hyperfine >/dev/null 2>&1 ]; then
+  echo "Please install hyperfine before running this script: https://github.com/sharkdp/hyperfine"
+  exit 1
+fi
+
+# Enable globstar for recursive directory listing
+# This is for Bash 4.0 and later, which does not have recursive globbing enabled by default
+shopt -s globstar
+
+# Read all files in the benchmarks directory to warm up the file system cache
+for i in {1..10}
+do
+  cat ../yjit-bench/benchmarks/**/*.rb > /dev/null
+done
+
+# Run the parser benchmarks
+# These benchmarks compare the performance of the Prism parser with the Sorbet parser without
+# any other parts of the type checker enabled.
+hyperfine \
+  --warmup=10 --export-json="prism_benchmarks/data/parser/$(date '+%Y-%m-%d')-$(git rev-parse --short HEAD).json" --parameter-list parser sorbet,prism \
+  "bazel-bin/main/sorbet --parser={parser} --stop-after=parser ../yjit-bench/benchmarks"
```

Comment:
Since `yjit-bench` isn't part of the test suite do we need to ensure it's parsing successfully before benchmarking or is it not worth it?

Reply:
Oh that's actually a great question. Um I think if there's a parsing error, that's fine (in fact at some point we should probably do some benchmarks on code with bad syntax). If the parsing just fails entirely (like the process exits with code 1) then I can try to print out an error with the message and ask the user to run the script again. Good call!

Reply:
I like that idea a lot!